contracts/LpSugar.vy

```diff
@@ -692,20 +692,15 @@ def _positions(
           positions.append(pos)
 
     else:
-      # fetch unstaked CL positions
+      # Fetch unstaked CL positions.
+      # Since we can't iterate over pools, offset and limit don't apply here.
+      # TODO: figure out a better way to paginate over unstaked positions.
       positions_count: uint256 = nfpm.balanceOf(_account)
 
       for pindex in range(0, MAX_POSITIONS):
         if pindex >= positions_count or pools_done >= _limit:
```

Comment:
if pool iteration doesn't apply here, is `pools_done >= _limit` still valid?

Reply:
Good call, refactored.